lib/resorts/tignes/resort.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "Tignes",
+  "url": {
+    "host": "https://bulletin.lumiplan.pro",
+    "pathname": "/displayAllNoMarge.php?&station=tignes&region=alpes&pays=france&lang=en&displayDomaine=4"
```

Comment:
Ideally we should have a `public_host` option here that overrides `host` for the "resort website" link